package-lock.json

```diff
@@ -16903,7 +16903,7 @@
         "parse-entities": "^2.0.0",
         "repeat-string": "^1.5.4",
         "state-toggle": "^1.0.0",
-        "trim": "0.0.3",
+        "trim": "0.0.1",
```

Comment:
That's probably because the script which forces dependencies to be resolved to a particular version hasn't run. It's probably worth it running that after adding the commits.

Apparently, running that script to force resolutions should be part of a pre-commit hook, not a pre-install script, that was my bad.